client/my-sites/site-settings/navigation.jsx

```diff
@@ -86,21 +85,11 @@ export default React.createClass( {
 							{ strings.discussion }
 					</NavItem>
 
-					{
-						( ( ! site.jetpack && config.isEnabled( 'manage/plans' ) ) ||
-							( site.jetpack && config.isEnabled( 'jetpack/google-analytics' ) ) ) &&
```

Comment:
Thanks for asking, I realize I haven't touched this in the PR description. 

Oh, yes we are concerned about Jetpack, of course. The thing is that:

* We don't need those checks anymore, because both `manage/plans` and `jetpack/google-analytics` are enabled for all environments.
* Analytics does work for Jetpack sites right now - it uses the corresponding module for that.

An obvious move would be to clean up these keys from the config and all other uses throughout Calypso, which I'm planning to do in a subsequent PR.